### ISSUE TITLE
Mismatch of accuracy

### ISSUE THREAD

**Issue by Tamil-Selvan-Murugesan** [TamilSelvan061296/DL-animal-10#1]
**Problem:**

The accuracy of the model during inference(data used for inferencing is same as the train data) is only 30-40% where as the accuracy of the model on the test set when training was around 85%. 

**Solutions tried:** 
- Tried to use the same preprocessing pipeline and also a less restricted version of the preprocessing.

The accuracy metric used is just the plain accuracy.

**Comment by Tamil-Selvan-Murugesan** [comment]
Potential Issue 1:

- I trained my model for just 1 epoch. I only saved and used the model which achieved 89% accuracy. But this instance of the model didn't go through the complete epoch. And that maybe the reason. 
- So, try using the model which is trained for 1 full epoch

**Comment by Tamil-Selvan-Murugesan** [comment]
So, I trained my model for 3 full epoch but the accuracy didn't improve during inferencing. There are few classes which are being predicted correctly by the model like squirrel and spider but it confuses between dogs, cows and horses. 

So, maybe, I need to use different pre-trained model or train the convolutional network parameters as well or come with a new architecture on my own. But this problem, I will take it up a little later as I am currently working on a different AI project.
